phone/src/locale/no.json

```diff
@@ -1,220 +1,283 @@
+      "NAVBAR_HISTORY": "History",
+      "NAVBAR_DIAL": "Ring",
+      "NAVBAR_CONTACTS": "Kontakter",
+      "NO_HISTORY": "Du har ingen samtalehistorikk"
+    },
+    "TWITTER": {
+      "FEEDBACK": {
+        "CREATE_PROFILE_SUCCESS": "Opprettet profilen din",
+        "CREATE_PROFILE_FAILURE": "Klarte ikke opprette profil",
+        "EDIT_PROFILE_SUCCESS": "Oppdaterte profilen din",
+        "EDIT_PROFILE_FAILURE": "Kunne ikke oppdatere profil",
+        "CREATE_FAILED": "Klarte ikke sende tweeten",
+        "RETWEET_EXISTS": "Du har allerede retweetet denne!"
+      },
+      "MESSAGES": {
+        "NEW_RETWEET_BROADCAST": "@{{ profile_name }} retweetet noen!",
+        "NEW_BROADCAST": "Ny tweet fra @{{ profile_name }}!",
+        "UNREAD_MESSAGES": "Du har {{ count }} nye tweets i tidslinjen din!"
+      },
+      "SUBMIT_IMAGE": "Legg til media",
+      "TWEET": "Tweet",
+      "REPORTED": " Rapportert",
+      "REPORT": "Rapporter",
+      "IMAGE_PLACEHOLDER": "En link til et bilde eller GIF",
+      "EDIT_DEFAULT_PROFILE_NAME": "Velg profilnavn",
+      "EDIT_PROFILE_AVATAR": "Avatar URL",
+      "EDIT_PROFILE_NAME": "Profilnavn",
+      "EDIT_PROFILE_BIO": "Bio",
+      "EDIT_PROFILE_LOCATION": "Lokasjon",
+      "EDIT_PROFILE_JOB": "Jobb",
+      "RETWEETED": "Retweetet",
+      "TWEET_MESSAGE_CHAR_LIMIT": "Meldingen er over karaktergrensen",
+      "TWEET_MESSAGE_NEW_LINE_LIMIT": "Meldingen er over linjegrensen",
+      "TWEET_MESSAGE_PLACEHOLDER": "Hva skjera brur?",
+      "SEARCH_TWEETS_PLACEHOLDER": "Søk etter melding eller profilnavn",
+      "SEARCH_TWEETS": "Søk tweets",
+      "TIME_JUST_NOW": "Akkuratt nå",
+      "TIME_MINUTES": "m",
+      "TIME_HOURS": "h",
+      "TIME_DAYS": "d"
+    },
+    "MATCH": {
       "MESSAGES": {
-        "INVALID_SETTINGS": "Invalid stored settings! Please reset in the Settings App.",
-        "SETTINGS_RESET": "Instillinger har blitt tilbakestilt"
+        "PROFILE_MATCHED_AT": "Matchet: {{ matchedAt }}",
+        "PROFILE_LAST_ACTIVE": "Sist Aktiv: {{ lastActive }}",
+        "LIKED": "LIKE",
+        "NOPE": "NEI"
+      },
+      "FEEDBACK": {
+        "UPDATE_PROFILE_FAILED": "En feil oppsto ved å opprette profil",
+        "UPDATE_PROFILE_SUCCEEDED": "Oppdaterte profil",
+        "MATCHES_ERROR": "Klarte ikke laste matches",
+        "NO_MATCHES": "Ingen matches funnet",
+        "NO_PROFILES": "Ingen nylige profiler funnet",
+        "PROFILES_ERROR": "Klarte ikke laste profiler",
+        "SAVE_LIKES_FAILED": "Klarte ikke lagre likes",
+        "NEW_LIKE_FOUND": "Ny match funnet!"
+      },
+      "PROFILE_CONFIGURATION": "Appen er feilkonfigurert - kan ikke opprette profil. Vennligst kontakt en administrator.",
+      "EDIT_PROFILE_IMAGE": "Bildeurl",
+      "EDIT_PROFILE_NAME": "Profilnavn",
+      "EDIT_PROFILE_BIO": "Bio",
+      "EDIT_PROFILE_LOCATION": "Lokasjon",
+      "EDIT_PROFILE_JOB": "Jobb",
+      "EDIT_PROFILE_TAGS": "Tagger",
```

Comment:
Would probably be "Knagger" lol, but if you want to keep it as "Tags", sure.

Reply:
"Tagger" sounds better imo, but yeah the direct translation would be "Knagger" as you said. What do you think?

Reply:
What about just "Emneknagg", too keep it more centralized. Then whomever that want to, can change it.

Reply:
Yeah I agree, will do.